2_process/src/process_data.R

```diff
@@ -0,0 +1,26 @@
+# Load necessary packages
+library(dplyr)
+library(stringr)
+
+# Function to process data
+process_data <- function(input_data, output_path) {
+  # load data
+  mendota_file <- readr::read_csv(input_data, col_types = 'iccd')
```

Comment:
as a variable name, perhaps `prediction_data` is more appropriate. That way you could input non-Mendota files in the future and have the variable names still make sense. 

Also, when you read in the data, "file" is probably a little misleading in the variable name.